tests/conftest.py

```diff
@@ -72,13 +75,27 @@ def pytest_addoption(parser):
         help="Run mini-benchmark tests with specified mode",
     )
 
+    parser.addoption(
+        "--bm-num-simulations",
+        action="store",
+        default=1000,
```

Comment:
default=2000 matches setting before. I think 2000 is a bit better because training would not be much longer but we would have less variance.